x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/network/dns/query.dns_network.dsl.ts

```diff
@@ -19,20 +18,20 @@ type QueryOrder =
   | { dns_bytes_in: { order: Direction } }
   | { dns_bytes_out: { order: Direction } };
 
-const getQueryOrder = (sort: SortField<NetworkDnsFields>): QueryOrder => {
-  switch (sort.field) {
-    case NetworkDnsFields.queryCount:
-      return { _count: { order: sort.direction } };
-    case NetworkDnsFields.dnsName:
-      return { _key: { order: sort.direction } };
-    case NetworkDnsFields.uniqueDomains:
-      return { unique_domains: { order: sort.direction } };
-    case NetworkDnsFields.dnsBytesIn:
-      return { dns_bytes_in: { order: sort.direction } };
-    case NetworkDnsFields.dnsBytesOut:
-      return { dns_bytes_out: { order: sort.direction } };
+const getQueryOrder = (sort: NetworkDnsRequestOptions['sort']): QueryOrder => {
+  if (sort.field === NetworkDnsFields.queryCount) {
```

Comment:
Is there a reason why this is cannot be a `switch/case` anymore? The `throw` could happen in a `default` case.